web/v1/redshift/sql-runner/sql/custom/01-indicative-export/02-indicative-user-stitching.sql

```diff
@@ -0,0 +1,33 @@
+-- Create user mapping tables for this run
+-- Also update the output table in preparation for joining to `indicative_export_staged` table
+
+CREATE TABLE {{.scratch_schema}}.user_stitching_this_run{{.entropy}} AS (
+
+  WITH users_this_run AS (
+    SELECT
+      domain_userid,
+      MIN(user_id) as user_id,
```

Comment:
Personally I would use `row_number()` here to select the first `user_id` per `domain_userid` rather than max it. If `events_staged` contains a large range of dates, i.e. it is a back fill of events data, then the decision of which `user_id` to choose is important.